client/src/app/pages/main/gamemodes/gamemodes.component.html

```diff
@@ -4,70 +4,51 @@
-        playerInterval="20000"
-        [items]="currentGamemodesItems"
-        [itemTemplate]="itemTemplate"
-        gallerize>
-      </gallery>
+  </div>
+  <div class="row justify-content-center mb-5 mb-lg-5">
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(0)" [ngClass]="(this.gameModeIndex == 0) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_skillsurf.png">
     </div>
-    <a href="{{this.currentGameMode.url}}" class="col-10 col-lg-10 mb-2 mb-lg-2">
-      <div class="display-5 text-center">
-        {{this.currentGameMode.modeTitle}}
-      </div>
-    </a>
-    <div class="col-10 col-lg-7 text-center text-white lead">
-      {{this.currentGameMode.modeDescription}}
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(1)" [ngClass]="(this.gameModeIndex == 1) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_bhop.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(2)" [ngClass]="(this.gameModeIndex == 2) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_rocketjump.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(3)" [ngClass]="(this.gameModeIndex == 3) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_stickyjump.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(7)" [ngClass]="(this.gameModeIndex == 7) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_climb.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <img width="100%" (click)="setCurrentGamemode(4)" [ngClass]="(this.gameModeIndex == 4) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_ahop.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <span class="gamemode-coming-soon">soon™</span>
+      <img width="100%" (click)="setCurrentGamemode(8)" [ngClass]="(this.gameModeIndex == 8) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_tricksurf.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <span class="gamemode-coming-soon">soon™</span>
+      <img width="100%" (click)="setCurrentGamemode(5)" [ngClass]="(this.gameModeIndex == 5) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_parkour.png">
+    </div>
+    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
+      <span class="gamemode-coming-soon">soon™</span>
+      <img width="100%" (click)="setCurrentGamemode(6)" [ngClass]="(this.gameModeIndex == 6) ? 'gamemode-icon-selected': 'gamemode-icon'" src="assets/images/gamemode_icons/site_conc.png">
```

Comment:
This is ripe for an [`ngFor` loop](https://angular.io/api/common/NgForOf#description). You can move the icon to the gamemode data array and do something like:
```html
<div class="row justify-content-center mb-5 mb-lg-5" *ngFor="let gamemode of gameModes; index as i">
    <div class="col-3 col-sm-3 col-md-2 col-lg-1">
      <img width="100%" (click)="setCurrentGamemode(index)" [ngClass]="(this.gameModeIndex == index) ? 'gamemode-icon-selected': 'gamemode-icon'" [src]="gamemode.iconURL">
    </div>
</div>
```

Reply:
should be resolved in c9eb97c